src/components/layers-panel/layers-panel.tsx

```diff
@@ -197,11 +197,33 @@ export const LayersPanel = ({
 
   useClickOutside([warningNode], () => setShowExistedError(false));
 
+  const convertUrlToRestFormat = (url: string): string => {
+    let urlRest = "https://www.arcgis.com/sharing/rest/content/items/";
+    const urlObject = new URL(url);
+
+    let param: string | null = null;
+    for (const paramName of ["id", "webscene", "layers"]) {
+      param = urlObject.searchParams.get(paramName);
+      if (param) {
+        break;
+      }
+    }
+    if (param) {
+      urlRest += param + "/data";
+    } else {
+      // The url cannot be converted. Use it "as is".
+      return url;
+    }
+    return urlRest;
+  };
+
   const handleInsertLayer = (layer: {
     name: string;
     url: string;
     token?: string;
   }) => {
+    layer.url = convertUrlToRestFormat(layer.url);
```

Comment:
Does it really work for layers? We were about doing it for scenes

Reply:
Removed.